homeassistant/components/twinkly/light.py

```diff
@@ -332,3 +339,22 @@ async def async_update_current_movie(self) -> None:
         _LOGGER.debug("Current movie: %s", current_movie)
         if current_movie and "id" in current_movie:
             self._current_movie = current_movie
+
+    async def async_update_current_color(self) -> None:
+        """Update the current active color."""
+        current_color = await self._client.get_current_colour()
+        _LOGGER.debug("Current color: %s", current_color)
+        if str(current_color.get("code")) == str(TWINKLY_RETURN_CODE_OK):
```

Comment:
If this is going to return a string, should the constant itself be stored as a string?

Reply:
I changed this quite a bit now.  Hopefully to something more generic, that can later be added to more of the code.
I did not want to add the validation to all the API-requests in this PR, since they are irellevant for the specific issue and it would make the PR much bigger.

If this validation-code is accepted, I'll wrap the rest of the API-calls in a later PR.